packages/amplify-category-function/src/events/prePushHandler.ts

```diff
@@ -23,3 +30,28 @@ const ensureFunctionSecrets = async (context: $TSContext) => {
   }
   await storeSecretsPendingRemoval(context, functionNames);
 };
+
+/**
+ * updates function cfn stack with lambda execution role arn parameter
+ */
+const ensureLambdaExecutionRoleOutputs = async (): Promise<void> => {
+  const amplifyMeta = stateManager.getMeta();
+  const functionNames = Object.keys(amplifyMeta?.[categoryName]);
+  for (const functionName of functionNames) {
+    const templateSourceFilePath = path.join(pathManager.getBackendDirPath(), categoryName, functionName, `${functionName}-cloudformation-template.json`);
+    const { cfnTemplate } = readCFNTemplate(templateSourceFilePath);
+    if (cfnTemplate?.Outputs?.LambdaExecutionRoleArn) {
+      return;
```

Comment:
should this return from the whole function or just go to the next iteration of the loop?